v22.2/use-cloud-storage-for-bulk-operations.md

```diff
@@ -253,12 +257,66 @@ Has permission to assume role A. See [step 2](#step-2-user). | Has a trust polic
+
+This means that a CockroachDB node will only be able to access credentials for the IAM role associated with the Kubernetes service account.
+
+You can use workload identities with assume role authentication to run the following operations:
+
+- [`BACKUP`](backup.html)
+- [`CREATE CHANGEFEED`](create-changefeed.html)
+- [`EXPORT`](export.html)
+- [`IMPORT`](import.html)/[`IMPORT INTO`](import-into.html)
+- [`RESTORE`](restore.html)
+
+There are multiple IAM roles referred to in this section, see the following descriptions:
+
+- _Identity role_: the IAM role you have associated with your Kubernetes service account.
+- _Operation role_: the IAM role to be assumed. This contains the permissions required to complete a CockroachDB operation.
+
+#### Set up AWS workload identity
+
+First of all, it is necessary to create an IAM role for your Kubernetes service account to assume, and then configure your CockroachDB pods to use the service account. We will refer to this IAM role as an "identity role". You can complete all of these steps with Amazon's guide on [IAM roles for service accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
+
+Once you have an identity role that your CockroachDB nodes can assume, you can configure the identity role to assume another IAM role that contains the permissions to perform a bulk operation. 
+
+1. Copy the ARN of identity role. In the Amazon management console, click on **IAM**, then **Roles**, and the name of your identity role. From the **Summary** page, copy your ARN. You will need this in the Trust Policy of the IAM role to be assumed.
+
+    <img src="{{ 'images/v22.2/aws-wi-arn-copy.png' | relative_url }}" alt="Role summary page showing the ARN copied" style="border:1px solid #eee;max-width:100%" />
+
+1. Create or open the IAM role (operation role) that your identity role will assume.
+
+    a. To create a role, click **Create Role** under the **Roles** menu. Select **Custom trust policy** and then add the ARN of your identity role to the JSON by clicking `Principal`. This will open a dialog box. Select **IAM Roles** for **Principal Type** and paste the ARN. Click **Add Principal** and then **Next**.
+
+    <img src="{{ 'images/v22.2/aws-wi-principal.png' | relative_url }}" alt="Dialog box to add principal with IAM roles selected" style="border:1px solid #eee;max-width:100%" />
+
+    b. On the **Add Permissions** page, search for the permission policies that the role will need to complete the bulk operation. Or, use the **Create Policy** button to define the required permissions. See [Storage Permissions](#storage-permissions) for detail on the minimum permissions required for each operation to complete. Click **Next**. 
```

Comment:
Question: We have screenshots for AWS' steps 1 (our step 2a) and 3 (our step 2c). No need for a screenshot for step 2 (our step 2b)? Because the user can branch here -- either "search for policies" or click `Create Policy` -- it might be helpful to visually show the UI for choosing.